x-pack/plugins/observability/public/components/app/cases/case_view/markdown_editor/plugins/lens/plugin.tsx

```diff
@@ -0,0 +1,386 @@
+/*
```

Comment:
Definitely not an issue for the first iteration, but would it make sense to move this out of the o11y plugin into the lens plugin to make it usable everywhere?

Reply:
Agree @flash1293, I do appreciate your comments, all are valid and I'm going to work on them in the next steps. We wanted first to make sure the UI works as expected

Could you point us to how to register migration of embedded lens configs?